### DISCUSSION TITLE
get input value in TagsInput suggestions closure

### DISCUSSION THREAD

**Discussion by IVoyt** [filamentphp/filament#13763] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

The closure in TagsInput/SpatieTagsInput suggestions method does not expect input value. Is there any possibility to get the input value in the closure?

**Comment by leandrocfe** [comment]
Could you explain what are you trying to do?

**Reply by IVoyt** [reply]
I want to fetch suggestions from backend when typing

**Reply by leandrocfe** [reply]
You need to create a custom field, I guess
